tests/link/jax/test_scan.py

```diff
@@ -13,11 +14,14 @@
 from pytensor.scan.op import Scan
 from pytensor.tensor import random
 from pytensor.tensor.math import gammaln, log
-from pytensor.tensor.type import lscalar, scalar, vector
+from pytensor.tensor.type import dmatrix, dvector, lscalar, scalar, vector
 from tests.link.jax.test_basic import compare_jax_and_py
 
 
-jax = pytest.importorskip("jax")
+# jax = pytest.importorskip("jax")
+
+
+jax.config.update("jax_platform_name", "cpu")
```

Comment:
This was left in by accident, please ignore.

I was getting errors when I ran the test suite on GPU. Google said it was a VSCode issue, but I'd be interested to know if others have this problem.